docs/data/currentVersions.json

```
@@ -32,6 +33,7 @@
       "appVersion": "2.14.5.0-b18",
       "isStable": true,
       "isLTS": true,
+      "isSTS": true,
```

A stable version can be either STS or LTS, but not both.
```suggestion
      "isSTS": false,
```